community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/planner/logical/CardinalityCostModel.scala

```diff
@@ -19,47 +19,69 @@
  */
 package org.neo4j.cypher.internal.compiler.v2_2.planner.logical
 
-import org.neo4j.cypher.internal.compiler.v2_2.ast.Collection
+import org.neo4j.cypher.internal.compiler.v2_2.ast.{HasLabels, Property, Collection}
 import org.neo4j.cypher.internal.compiler.v2_2.commands.ManyQueryExpression
 import org.neo4j.cypher.internal.compiler.v2_2.planner.logical.Metrics._
 import org.neo4j.cypher.internal.compiler.v2_2.planner.logical.plans._
 
-/*
-A very simplistic cost model. Each row returned by an operator costs 1. That's it.
- */
 case class CardinalityCostModel(cardinality: CardinalityModel) extends CostModel {
 
-  val CPU_BOUND_PLAN_COST_PER_ROW: CostPerRow = 0.1
-  val DB_ACCESS_BOUND_PLAN_COST_PER_ROW: CostPerRow = 1.0
+  val CPU_BOUND:  CostPerRow = 0.1
+  val FAST_STORE: CostPerRow = 1.0
+  val SLOW_STORE: CostPerRow = 10.0
```

Comment:
missing `private`?

Moreover it would be nice for future reference to explain how these constants have been computed and why the ended up having the current values.